src/client.rs

```diff
@@ -50,11 +51,34 @@ impl Client {
         //     });
         // }
         // Ok(Channel::balance_list(endpoints.into_iter()))
-        let end_points = cfg.endpoints.join(",");
+        if cfg.endpoints.is_empty() {
+            panic!("Empty etcd endpoints");
+        }
+        let mut end_points = cfg.endpoints.join(",");
         let env = Arc::new(EnvBuilder::new().build());
+        if cfg.endpoints.len() > 1 {
+            let socket_address: SocketAddr = cfg
+                .endpoints
+                .first()
```

Comment:
only accept first etcd address?

Reply:
No, it will only use the first address to check the schema, not only accept first etcd address